packages/core/src/App/Containers/Layout/header/traders-hub-header.jsx

```diff
@@ -305,66 +212,52 @@ const TradingHubHeader = ({
     );
 };
 
-TradingHubHeader.propTypes = {
+TradersHubHeader.propTypes = {
+    acc_switcher_disabled_message: PropTypes.string,
+    account_type: PropTypes.string,
+    balance: PropTypes.string,
+    country_standpoint: PropTypes.object,
+    currency: PropTypes.string,
+    has_any_real_account: PropTypes.bool,
     header_extension: PropTypes.any,
+    is_acc_switcher_disabled: PropTypes.bool,
+    is_acc_switcher_on: PropTypes.bool,
     is_app_disabled: PropTypes.bool,
     is_dark_mode: PropTypes.bool,
-    is_eu_country: PropTypes.bool,
     is_eu: PropTypes.bool,
     is_logged_in: PropTypes.bool,
     is_mt5_allowed: PropTypes.bool,
-    is_notifications_visible: PropTypes.bool,
     is_route_modal_on: PropTypes.bool,
     is_settings_modal_on: PropTypes.bool,
-    loginid: PropTypes.string,
+    is_virtual: PropTypes.bool,
     modal_data: PropTypes.object,
-    notifications_count: PropTypes.number,
     platform: PropTypes.string,
-    setIsOnboardingVisited: PropTypes.func,
     settings_extension: PropTypes.array,
-    toggleIsTourOpen: PropTypes.func,
-    toggleNotifications: PropTypes.func,
-    acc_switcher_disabled_message: PropTypes.string,
-    account_type: PropTypes.string,
-    balance: PropTypes.string,
-    currency: PropTypes.string,
-    is_acc_switcher_disabled: PropTypes.bool,
-    country_standpoint: PropTypes.object,
-    is_acc_switcher_on: PropTypes.bool,
-    is_virtual: PropTypes.bool,
     toggleAccountsDialog: PropTypes.func,
-    has_any_real_account: PropTypes.bool,
-    toggleReadyToDepositModal: PropTypes.func,
     toggleNeedRealAccountForCashierModal: PropTypes.func,
+    toggleReadyToDepositModal: PropTypes.func,
 };
 
-export default connect(({ client, common, notifications, ui, traders_hub }) => ({
+export default connect(({ client, common, ui, traders_hub }) => ({
```

Comment:
can't we use useStore in this component?

Reply:
sure.. migrating to ts.. 🚚